### ISSUE TITLE
Creating Installation Screencast of project and upload on youtube

### ISSUE THREAD

**Issue by sk9331657** [publiclab/community-toolbox#69]
Create a screencast showing the installation of the project and share the link in pr.

**Comment by Rishabh570** [comment]
I think the installation for this project is not that complex...but we can add a GIF in the readme anyways :)

**Comment by sk9331657** [comment]
@jywarren  @SidharthBansal  Please add GCI task tag.

**Comment by Rishabh570** [comment]
@jywarren Can it be converted into a GCI task?

**Comment by jywarren** [comment]
Yep! Doing so now.

**Comment by jywarren** [comment]
https://codein.withgoogle.com/dashboard/tasks/4761125351587840/

**Comment by oorjitchowdhary** [comment]
Which project? @publiclab/mentors

**Comment by gauravano** [comment]
Screencast of plots2

**Comment by gauravano** [comment]
Actually, we can create these tasks for all our active repositories.

**Comment by JonathanXu1** [comment]
Hi, I've claimed this task. Here's the screencast for the installation of the repository: https://youtu.be/PLPsm8K3Tl0

**Comment by JonathanXu1** [comment]
Here's the properly filmed screen cast: https://youtu.be/OLiyShl6Lh8

**Comment by gauravano** [comment]
@jywarren check this out. Also, any thoughts about the next steps with the video :smiley: .

**Comment by Rishabh570** [comment]
@jywarren Do we want to have installation screencast for Linux, mac as well as windows?

**Comment by Faizan-Alam-1** [comment]
Is This issue solved?

**Comment by gauravano** [comment]
> Maybe we can mention that this installation works for every OS ?

 Fair point @Rishabh570 as installation steps would be similar for Ubuntu and Mac OS too (assumes npm, and Git are already installed). 

@jywarren if you also believe the same, this issue can be closed. Also, the video guide is already present at https://github.com/publiclab/community-toolbox/blob/main/docs/getting_started.md#installation-instructions--watch-video-guide-

Thank you for showing interest @Faizan-Alam-1. You can check out other open issues in the repo until we update status here.

Thanks again!!

**Comment by itsgurmannatsohal** [comment]
Is this issue open? Can I work on it?
